compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -2013,6 +2013,20 @@ TR_BitVector *OMR::SymbolReferenceTable::getSharedAliases(TR::SymbolReference *s
    return NULL;
    }
 
+
+TR::SymbolReference *
```

Comment:
I have added the findOrCreate for this type of symbols. These symbols are similar to InlinedByCodeGen symbols which will be inlined by the codegen (https://github.com/eclipse/omr/blob/8acafad8398cc9a596ba7d4f0821e18853711803/compiler/compile/OMRSymbolReferenceTable.cpp#L766) . Delta here would be that these will be lowered to trees by optimizer.

Reply:
passing in the index opens this API to misuse - the convention is one per special helper so I think that would make more sense and keep the code more terse...

Reply:
I had `FATAL_ASSERT` to prevent misuse. But following the convention here, separated out the functions for both symRef in  https://github.com/eclipse/omr/pull/4172/commits/619a159a143a9124a51498c9b209f18ccc5d4029
Let me know changes are fine. Will squash the commits.

Reply:
looks good to me

Reply:
Thanks a lot.. once builds are finished, will squash commits.